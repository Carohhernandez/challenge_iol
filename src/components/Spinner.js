import React from 'react';
import styled from 'styled-components';

const SpinnerContainer = styled.div`
    display: flex;
    justify-content: center;

    padding-top: 25px;

    i {
        color: #fff;
        font-size: 70px;
    }
`;

const Spinner = ({ color }) => {
    return (
        <SpinnerContainer>
            <i className="fa fa-refresh fa-spin fa-fw" aria-hidden="true" />
        </SpinnerContainer>
    );   
};

export default Spinner;