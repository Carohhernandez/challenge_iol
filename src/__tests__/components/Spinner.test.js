import { render, cleanup } from "@testing-library/react";
import Spinner from '../../components/Spinner';

afterEach(cleanup);

test("renders Spinner correctly", () => {
  const { asFragment } = render(<Spinner />);
  expect(asFragment()).toMatchSnapshot();
});