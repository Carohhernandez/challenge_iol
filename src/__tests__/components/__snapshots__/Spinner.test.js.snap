// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Spinner correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bcXHqe bYdSAJ"
  >
    <i
      aria-hidden="true"
      class="fa fa-refresh fa-spin fa-fw"
    />
  </div>
</DocumentFragment>
`;
